SPECS/kernel-mshv/50_mariner_mshv.cfg

```diff
@@ -1,16 +1,5 @@
-requirements="Windows/System32/hvix64.exe Windows/System32/hvax64.exe lxhvloader.dll HvLoader.efi"
-for file in $requirements; do
```

Comment:
Given that this is a azl3 image and therefore generated by grub2-mkconfig, I think this config should stay as it is for 3.0.

Reply:
Good point. Rebased against 3.0-dev branch.